cvmfs/mountpoint.h

```diff
@@ -469,6 +469,10 @@ class MountPoint : SingleCopy, public BootFactory {
   void CreateFetchers();
   bool CreateCatalogManager();
   void CreateTables();
+  /**
```

Comment:
We should have only one comment for the method, in the .cc file.  This keeps the header quickly digestible and the comments all in one place.